tests/widgets/test_datasetInfoEditorWidget.py

```diff
@@ -0,0 +1,227 @@
+import os
+from typing import List, Tuple
+from numbers import Number
+from pathlib import Path
+import pytest
+import requests
+import tempfile
+import uuid
+import h5py
+
+import numpy
+from PyQt5.QtCore import Qt
+
+from ilastik.applets.dataSelection.datasetInfoEditorWidget import DatasetInfoEditorWidget
+from ilastik.applets.dataSelection.dataSelectionSerializer import DataSelectionSerializer
+from ilastik.applets.dataSelection.opDataSelection import OpDataSelectionGroup
+from ilastik.applets.dataSelection import DatasetInfo
+from lazyflow.graph import Graph
+
+def download_test_image(url, suffix:str):
+    resp = requests.get(url)
+    _, image_path = tempfile.mkstemp(suffix='-' + suffix)
+    with open(image_path, 'wb') as f:
+        f.write(resp.content)
+    return image_path
+
+@pytest.fixture(scope='function')
+def image_yxc_path():
+    path =  download_test_image("http://data.ilastik.org/pixel-classification/2d/c_cells_1.png", "c_cells_1.png")
```

Comment:
Usual tests should not hit non-localhost networks. Such tests should be marked as "slow" or "network", and it should be possible to disable them (similarly to GUI tests).

Reply:
yep, maybe just see wether you can use something in `tests/data` or add something if needed

Reply:
I was kinda desperate for some data at the time of writing those tests and forgot to move those into a more sensible version, with images being generated on-the-fly. Fixed =)

Reply:
I was kinda desperate for some data at the time of writing those tests and forgot to move those into a more sensible version, with images being generated on-the-fly. Fixed =)